optee-packages.mk

```diff
@@ -0,0 +1,53 @@
+OPTEE_PLATFORM ?= poplar
+OPTEE_PLATFORM_FLAVOR ?= poplar
+OPTEE_CFG_ARM64_CORE ?= y
+OPTEE_TA_TARGETS ?= ta_arm32
+OPTEE_OS_DIR ?= optee/optee_os
+OPTEE_EXTRA_FLAGS ?= CFG_TEE_CORE_LOG_LEVEL=3 CFG_TEE_TA_LOG_LEVEL=3 DEBUG=1
+CROSS_COMPILE64 ?= $(ANDROID_BUILD_TOP)/prebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/bin/aarch64-linux-android-
+BUILD_OPTEE_MK := $(OPTEE_OS_DIR)/mk/aosp_optee.mk
```

Comment:
I don't know what is inside of aosp_optee.mk. But have to specify the mk file looks unusual on Android platform. Will rename `aosp_optee.mk` to  `Android.mk` work? so that we don't need to add a variable called BUILD_OPTEE_MK. Probably, you have a few .mk so it can work on other platforms as well.